spec.html

```diff
@@ -14027,9 +14870,13 @@ <h1>Runtime Semantics: Evaluation</h1>
         1. Let _lval_ be ? GetValue(_lref_).
         1. Let _rref_ be the result of evaluating _B_.
         1. Let _rval_ be ? GetValue(_rref_).
-        1. Let _lnum_ be ? ToInt32(_lval_).
-        1. Let _rnum_ be ? ToInt32(_rval_).
-        1. Return the result of applying the bitwise operator @ to _lnum_ and _rnum_. The result is a signed 32-bit integer.
+        1. Let _lnum_ be ? ToNumeric(_lval_).
+        1. Let _rnum_ be ? ToNumeric(_rval_).
+        1. If Type(_lnum_) is different from Type(_rnum_), throw a *TypeError* exception.
+        1. Let _T_ be Type(_lnum_).
+        1. If @ is `&amp;`, return _T_::bitwiseAND(_lnum_, _rnum_).
```

Comment:
this is kind of what i'd expect for the `_op_()` i commented on above

Reply:
Yes. I agree. We need to define some Abstract Operations to properly use `_op_()` there.